### DISCUSSION TITLE
find index in enums by value

### DISCUSSION THREAD

**Discussion by hoangtraann** [community/community#65155] [Programming Help]
### Body

How i can find index in enums by value? in nodejs 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by nhh0718** [comment]
const findEnumIndexByValue = (enumObject, targetValue) =>
Object.values(enumObject).indexOf(targetValue);
In nodejs you can do this, try.

**Reply by hoangtraann** [reply]
thank you, this help

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
